bulbs/special_coverage/filters.py

```diff
@@ -18,9 +18,13 @@ def filter_queryset(self, request, queryset, view):
             if value == "true":
                 queryset = queryset.filter(
                     start_date__lte=today_filter, end_date__gte=today_filter
+                ) | queryset.filter(
+                    start_date__lte=today_filter, end_date__isnull=True
```

Comment:
we _could_ do something like a custom manager query so we get 

`SpecialCoverage.objects.active()` rather than re-writing this a bunch of places, but that's mostly personal preference